frontends/qt/libserver.h

```diff
@@ -22,6 +22,14 @@ typedef char* (*getSaveFilenameCallback) (const char*);
 static char* getSaveFilename(getSaveFilenameCallback f, const char* suggestedfilename) {
     return f(suggestedfilename);
 }
+
+// equivalent to C.free but suitable for releasing a memory malloc'ed
+// in a non-posix portable environment, incompatible with cgo.
+// this is especially important on windows where the standard C runtime
+// memory management used by cgo and mingw is different from win32 API and
+// hence the memory malloc'ed by the "new" operator in C++ or HeapAlloc must
```

Comment:
why "new" operator? we use `malloc()` to put the string into the heap.

Reply:
because both `new ...` and `malloc` would use the same heap in that memory world. just wanted to put more emphasis on distinguishing portable C runtime's `malloc` and "the other" `malloc`, and using only `malloc` as an example everywhere seemed more confusing.

as a matter of fact, these two allocate same amount of memory on the same heap in c++:

```c
a = (char*) malloc(10);
b = new char[10];
```

but because the `new` operator also calls a class constructor, when it's allocating memory for objects, you made me realize now i've probably made things more confusing. will remove the "new" operator.